packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -148,73 +104,28 @@ export class EditorPage {
 
 	/**
 	 * Initialization steps to ensure the page is fully loaded.
-	 *
-	 * @returns {Promise<Frame>} iframe holding the editor.
 	 */
 	async waitUntilLoaded(): Promise< void > {
-		const timeout =
-			this.target === 'atomic' ? EXTENDED_EDITOR_WAIT_ATOMIC_TIMEOUT : EXTENDED_EDITOR_WAIT_TIMEOUT;
-
 		// In a typical loading scenario, this request is one of the last to fire.
 		// Lacking a perfect cross-site type (Simple/Atomic) way to check the loading state,
 		// it is a fairly good stand-in.
 		await Promise.all( [
 			this.page.waitForURL( /(post|page|post-new.php)/ ),
-			this.page.waitForResponse( /.*posts.*/, { timeout: timeout } ),
+			this.page.waitForResponse( /.*posts.*/, { timeout: 60 * 1000 } ),
```

Comment:
**question:non-blocking** - is 60s going to be long enough for AT loading, where it tries to load the iframed version then redirects to the "raw" wp-admin based edtior?

If it is, would it make sense to export the `EditorComponent.EDITOR_TIMEOUT` value so that we have one defined value for the loading timeout?

Reply:
> is 60s going to be long enough for AT loading, where it tries to load the iframed version then redirects to the "raw" wp-admin based editor?

[60s is the value we previously used specifically for the AT editor timeout](https://github.com/Automattic/wp-calypso/pull/76085/files/48843472678df6810801eadad3a97b775146a060#diff-9a431df4f005592b6887e18be632044cbc38b331da80fc73f9a0c9fa7efcc5beL44-L45), so it should be good at this point.

> If it is, would it make sense to export the EditorComponent.EDITOR_TIMEOUT value so that we have one defined value for the loading timeout?

Ultimately, I think we shouldn't need that timeout value to be used outside the `EditorComponent`. Waiting for the editor parent/canvas should be all we need to verify that they're ready to be interacted with (thinking optimistically 😛). For example, the above should be replaceable with `await this.editor.parent()`. I didn't want to do that in this PR since it requires a separate round of testing, though. If that's OK, I'd keep it as is and I'll address this bit in a follow-up PR.